app/ui-react/syndesis/src/modules/integrations/components/editor/utils.ts

```diff
@@ -42,6 +44,44 @@ type StepKindHrefCallback = (
   s: ISelectConnectionRouteState
 ) => H.LocationDescriptorObject;
 
+/**
+ * Checks whether or not the provided object is a technical preview.
+ * Accepts either an IUIStep, Connection, or a Connector.
+ * Returns a boolean for whether or not the metadata `tech-preview` key
+ * returns a string value of 'true'
+ * @param data
+ */
+export function figureOutTechPreviewFlag<
```

Comment:
Probably `isTechPreview` would be better, I know I said `figureOutTechPreviewFlag` but that was being a bit tongue-in-cheek, sorry!.

Reply:
haha no worries, I thought so, but either way we won't be able to use the same declaration so maybe `isTechPreviewCheck` might work better? Or something similar?